commit 396be5abfb4bcabe30096c21b0509227106e1137
Author: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date:   Mon Feb 3 03:04:32 2025 +0000

    Update elements/include/linux.yml to v6.13.1-0
    
    Updates elements/include/linux.yml to v6.13.1-0 (16dc8d5)
    Previous version was v6.13-0 (ffd294d)

diff --git b/elements/include/linux.yml a/elements/include/linux.yml
index 1e760a4a9..05dff1f9f 100644
--- b/elements/include/linux.yml
+++ a/elements/include/linux.yml
@@ -4,7 +4,7 @@ sources:
   track: v*
   exclude:
   - '*-rc*'
-  ref: v6.13.1-0-g16dc8d5d6fc132f09b3fa6021f94f11c60e39957
+  ref: v6.13-0-gffd294d346d185b70e28b1a28abe367bbfe53c04
 - kind: local
   path: files/linux/config-utils.sh
 - kind: local
